### PR TITLE
Adds the medbeam module which replaces the Nukie Medgun

### PR DESCRIPTION
**Pull request by DaydreamIQ**
## About The Pull Request
Adds the Medbeam module, essentially works the same way as the implant version does where it pops into your hand. Replaces the handheld version with the modsuit one in the nuclear uplink
## Why It's Good For The Game
Ever since modsuits were added I've wanted to see this become a module of its own, that and the medbeam itself doesn't see much use in most nukies rounds. As the borg gets that plus the nanities with nodrop, this might give it a teeny bit more use cases as now the operatives won't have to worry about dropping it in the middle of a firefight.
## Changelog
:cl:
add: Adds the medbeam module for nukies, don't cross the damn beams. (Also removes the handheld one from the uplink)
/:cl:

### CODE REVIEW

**Comment by Helg2** [comment]
You could pretty much also replace it for ert.
Also changelog doesn't describe all the change and can be misleading.

File: code/modules/uplink/uplink_items/nukeops.dm [hunk 1]
```diff
@@ -597,6 +597,14 @@
 	cost = 2
 	purchasable_from = UPLINK_NUKE_OPS | UPLINK_CLOWN_OPS
 
+	/datum/uplink_item/device_tools/medgun_mod
```
Comment (Fikou) [inline code/modules/uplink/uplink_items/nukeops.dm:600]:
```suggestion
/datum/uplink_item/device_tools/medgun_mod
```

**Comment by DaydreamIQ** [comment]
> You could pretty much also replace it for ert. Also changelog doesn't describe all the change and can be misleading.

I wanted to but it'd mean either slapping a module in their bag with a screwdriver, or giving it to all alert levels of ERT. When I get a better idea of what im doing i'll probably come back to it.

**Comment by Jackal-boop** [comment]
the syndicate modsuit already doesnt have enough space to install all of the modules that come with the nukies intended healer medical suite, i dont think adding another one is a good thing

**Comment by SimplySmiley123** [comment]
> the syndicate modsuit already doesnt have enough space to install all of the modules that come with the nukies intended healer medical suite, i dont think adding another one is a good thing

This. Modsuits are filled by default with not much space left for whatever you buy.

**Comment by Rex9001** [comment]
I really like the idea of this pr, but as mentioned you already need to take out modules to fit in very simple ones into the nukie modsuit. Id suggest maybe bumping the complexity up a notch so this isnt just a nukie nerf?

**Comment by DaydreamIQ** [comment]
I put it at 2 cause that seemed to be the standard for this kind of module, but if needs be I can drop it to 1.

**Review by Ghommie — APPROVED** [review]
No-drop medbeams: the thread. I suppose this is fairly justified since the item costs 15 tcs anyway.
EDIT: Also, yeah, the complexity limit could be a bit worrying in the long run.

**Pull request merged**